package.json

```diff
@@ -11,9 +11,12 @@
     "format": "prettier --write src/"
   },
   "dependencies": {
+    "@types/jwt-decode": "^3.1.0",
```

Comment:
zover ik zie is jwt-decode deprecated dus kan je beter wat anders gebruiken, je hoeft voor dit sowieso niet de jwt te decoden